projects/plugins/jetpack/sal/class.json-api-post-base.php

```diff
@@ -209,41 +391,68 @@ public function get_other_urls() {
 		return (object) $other_urls;
 	}
 
+	/**
+	 * Calls the WPCOM_JSON_API_Links get_site_link() function to generate a site link endpoint URL.
```

Comment:
Here's I'd skip the "Calls the WPCOM_JSON_API_Links get_site_link() function to" bit and just say what it does. Unless the implementation is important the caller probably doesn't need to know the "how".